src/Metadata/HttpOperation.php

```diff
@@ -55,6 +55,7 @@ class HttpOperation extends Operation
      *     stale_while_revalidate?: int,
      *     stale-if-error?: int,
      * }|null $cacheHeaders {@see https://api-platform.com/docs/core/performance/#setting-custom-http-cache-headers}
+     * @param array<string, string>|null $headers
```

Comment:
```suggestion
     * @param array<string, mixed>|null $headers
```

Reply:
Headers are basically strings (https://httpwg.org/specs/rfc9110.html#fields.values) it looks like the HeaderBag of symfony also only types strings so I should rather enforce a string here no?